### PR TITLE
CTAA_1386 exposure keys dowload API

### PR DESCRIPTION
**Pull request by falkorichter**
API with debug screen

## Changes

API scaffold to get the exposure keys index with debug UI for testing testing

new properties: 
- `STAGE_BASE_URL_CDN`
- `STAGE_HOSTNAME_CDN`
- `PROD_BASE_URL_CDN`
- `PROD_HOSTNAME_CDN`

## Solved issues

- [Issue #CTAA_1386](https://tasks.pxp-x.com/browse/CTAA-CTAA_1386)

### CODE REVIEW

File: app/src/main/java/at/roteskreuz/stopcorona/model/entities/infection/exposure_keys/IndexOfExposureKeysArchive.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Describes infection info about user with data gathered from the Exposure SDK.
+ */
+@JsonClass(generateAdapter = true)
+data class IndexOfExposureKeysArchive(
+    @field:Json(name = "full_batch")
+    val full_batch: Batch,
```
Comment (JU5T1C3) [inline app/src/main/java/at/roteskreuz/stopcorona/model/entities/infection/exposure_keys/IndexOfExposureKeysArchive.kt]: please refactor to camelCase

File: app/src/main/java/at/roteskreuz/stopcorona/screens/debug/exposure_notification_infection_messages/DebugExposureNotificationsTrackingKeysFragment.kt [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        }
+    }
+
+    override fun onResume() {
+        super.onResume()
+        viewModel.checkEnabledState()
+    }
+
+    override fun getTitle(): String? {
+        return "Exposure Tracing Tracking Keys"
```
Comment (JU5T1C3) [inline app/src/main/java/at/roteskreuz/stopcorona/screens/debug/exposure_notification_infection_messages/DebugExposureNotificationsTrackingKeysFragment.kt]: please don't use the word `Tracing`

File: app/src/main/java/at/roteskreuz/stopcorona/model/entities/infection/exposure_keys/IndexOfExposureKeysArchive.kt (hunk 1 above)
Comment (dusanjencik) [inline app/src/main/java/at/roteskreuz/stopcorona/model/entities/infection/exposure_keys/IndexOfExposureKeysArchive.kt]: Please rename to `fullBatch`

**Comment by JU5T1C3** [comment]
🆗

**Review by dusanjencik — DISMISSED** [review]
Some debug screen ok. Let's merge if it help you. When will be real implementation and connection?

👍

**Review by JU5T1C3 — APPROVED** [review]

**Pull request merged**